config/config.txt

```diff
@@ -320,12 +320,16 @@ ANNOUNCE_ADMIN_LOGOUT
 
 ## Map rotation
 ## You should edit maps.txt to match your configuration when you enable this.
-MAPROTATION
+## Set to 0 for no map rotation, 1 for pref-based maprotation, 2 for vote-based maprotation, 3 for random.
+MAPROTATION 2
 
-## Map voting
-## Allows players to vote for their preffered map
-## When it's set to zero, the map will be randomly picked each round
-ALLOW_MAP_VOTING 1
+## Map Rotation result method
+## Set to 1 for greatest, 2 for weighted random.
+MAPROTATION_RESULT_METHOD 2
```

Comment:
If there's only two options it should be a flag and it should state the default behaviour